src/dmg2john.c

```diff
@@ -110,6 +121,60 @@ static void print_hex(unsigned char *str, int len)
 		printf("%02x", str[i]);
 }
 
+// Remove these strnXYZ functions after refactoring misc.c
```

Comment:
This code duplication is not great. At least explain in the comment what's wrong with misc.c.